### PR TITLE
Added AfterPay countries available.

### PR DESCRIPTION
**Pull request by LeasoftKonstantinShapoval**
Fixed IP-filtering and Payment Method availability Zone method in AfterPay and KlarnaPayLater.

### CODE REVIEW

File: includes/classes/class.emspayGateway.php [hunk 1]
```diff
@@ -66,10 +66,12 @@ function __construct()
...
             $this->update_status();
             if ($this->isKlarnaPayLater()) {
-                $this->enabled = $this->emsKlarnaPayLaterIpFiltering();
+                $this->IsEnable($this->emsKlarnaPayLaterIpFiltering());
             }
             if ($this->isAfterPay()) {
-                $this->enabled = $this->emsAfterPayIpFiltering();
+                $this->IsEnable($this->emsAfterPayIpFiltering());
+                $this->IsEnable($this->emsAfterPayCountriesValidation($order));
+
```
Comment (tiutiunnykleasoft) [inline includes/classes/class.emspayGateway.php:74]: Remove empty string

File: includes/classes/class.emspayGateway.php [hunk 2]
```diff
@@ -66,10 +66,12 @@ function __construct()
         if (is_object($order)) {
             $this->update_status();
             if ($this->isKlarnaPayLater()) {
-                $this->enabled = $this->emsKlarnaPayLaterIpFiltering();
+                $this->IsEnable($this->emsKlarnaPayLaterIpFiltering());
             }
             if ($this->isAfterPay()) {
-                $this->enabled = $this->emsAfterPayIpFiltering();
+                $this->IsEnable($this->emsAfterPayIpFiltering());
+                $this->IsEnable($this->emsAfterPayCountriesValidation($order));
```
Comment (tiutiunnykleasoft) [inline includes/classes/class.emspayGateway.php:73]: If the "IP filter" option turns off the payment method, the country validation turns it to on. But on the logic of "IP Filter" payment method should be turned off. 
You should create a variable that should be true only if both statements return true.

Comment (tiutiunnykleasoft) [inline includes/classes/class.emspayGateway.php:73]: $this->enabled = $this->emsAfterPayIpFiltering() && $this->emsAfterPayCountriesValidation($order);

File: includes/classes/class.emspayGateway.php [hunk 3]
```diff
@@ -66,10 +66,12 @@ function __construct()
         if (is_object($order)) {
             $this->update_status();
             if ($this->isKlarnaPayLater()) {
-                $this->enabled = $this->emsKlarnaPayLaterIpFiltering();
+                $this->IsEnable($this->emsKlarnaPayLaterIpFiltering());
```
Comment (tiutiunnykleasoft) [inline includes/classes/class.emspayGateway.php:69]: $this->enabled = $this->enabled ? $this->emsKlarnaPayLaterIpFiltering() : false;

File: includes/classes/class.emspayGateway.php [hunk 4]
```diff
@@ -155,6 +157,19 @@ public function check()
...
+     *
+     * @return boolean
+     */
+    public function IsEnable($filter)
+    {
+        if ($this->enabled === true) {
+            return $this->enabled = $filter;
+        }
+    }
+
```
Comment (tiutiunnykleasoft) [inline includes/classes/class.emspayGateway.php:172]: Remove this function

File: includes/classes/class.emspayGateway.php [hunk 5]
```diff
@@ -155,6 +157,19 @@ public function check()
         return $this->_check;
     }
 
+    /**
+     * Check to if payment module is enable
+     * By default is $this->enabled = true
+     *
+     * @return boolean
+     */
+    public function IsEnable($filter)
```
Comment (tiutiunnykleasoft) [inline includes/classes/class.emspayGateway.php:166]: Remove this function

File: includes/classes/class.emspayGateway.php (hunk 1 above)
Comment (LeasoftKonstantinShapoval) [reply includes/classes/class.emspayGateway.php:74]: Done.

File: includes/classes/class.emspayGateway.php (hunk 2 above)
Comment (LeasoftKonstantinShapoval) [reply includes/classes/class.emspayGateway.php:73]: Changed.

File: includes/classes/class.emspayGateway.php (hunk 3 above)
Comment (LeasoftKonstantinShapoval) [reply includes/classes/class.emspayGateway.php:69]: Changed. Tested.

File: includes/classes/class.emspayGateway.php (hunk 5 above)
Comment (LeasoftKonstantinShapoval) [reply includes/classes/class.emspayGateway.php:166]: Done.

**Pull request merged**